dnfdaemon-client/main.cpp

```diff
@@ -93,6 +93,14 @@ static bool parse_args(Context & ctx, int argc, char * argv[]) {
     allow_erasing->link_value(ctx.allow_erasing.get());
     dnfdaemon_client->register_named_arg(allow_erasing);
 
+    auto skip_broken = ctx.arg_parser.add_new_named_arg("skip_broken");
+    skip_broken->set_long_name("skip-broken");
```

Comment:
Is there an approach how can I set `false`?

Reply:
This would require a new switch with inverted meaning - e.g. `--strict`. `skip-broken` is `true` when present on the command line, `false` otherwise.